cernopendata/modules/fixtures/data/records/opera-ecc-datasets.json

```diff
@@ -413,5 +413,123 @@
     "validation": {
       "description": "During the data taking, all the runs recorded by OPERA are certified as good for physics analysis if the trigger and all sub-detectors show the expected performance. Moreover, the time stamp of the event should lie within the gate open by the CNGS beam signal. The data certification is based first on the offline shifters evaluation and later on the feedback provided by all sub-detector experts. Based on the above information, stored in a specific database, the Data Quality Monitoring group verifies the consistency of the certification and prepares an ascii file of certified runs to be used for physics analysis. For this specific data record, dedicated calibration procedures are performed to align the emulsion films each other and with the electronic detectors. These procedures with the corresponding results are saved in a dedicated database where data quality experts certify the results and prepare files to be used for the track and vertex reconstruction, thus being available for physics analysis."
     }
+  },
+  {
+    "abstract": {
+      "description": "<p>FIXME description</p><p></p><p></p><p></p>"
```

Comment:
@galatigiuliana Everything is ready except for nu_e dataset descriptions. Do you have some text that I could use for the initial version? Or shall we merge and you'll provide descriptions later?